components/prism-bash.js

```diff
@@ -102,7 +102,7 @@
 		// Highlight variable names as variables in the left-hand part
 		// of assignments (“=” and “+=”).
 		'assign-left': {
-			pattern: /(^|[\s;|&]|[<>]\()\w+(?=\+?=)/,
+			pattern: /(^|[\s;|&]|[<>]\()(\w+\.)*\w+(?=\+?=)|(-{1,2})\w+(?=(\+?=|\s+|$))/,
```

Comment:
`assign-left` isn't intended for parameters. Please create a new token for command parameters. Also, why the `(\w+\.)*\w+`? Is `foo.bar=4` actually valid? (I basically never use bash.)

About the regex itself:

1. Only use capturing groups when necessary.
2. By using `\w+(?:\.\w+)*` we can cut down on backtracking, making the regex faster.
3. We should also support `--foo-bar=baz`.
```suggestion
			pattern: /(^|[\s;|&]|[<>]\()\w+(?:\.\w+)*(?=\+?=)|-{1,2}\w+(?:-\w+)*(?=\+?=|\s+|$)/,
```

Reply:
>     1. Only use capturing groups when necessary.

Ok.

>     2. By using `\w+(?:\.\w+)*` we can cut down on backtracking, making the regex faster.

Ok.

>     3. We should also support `--foo-bar=baz`.
 
The `=xxx` is not necessary.